software/spmd/interrupt_tests/Makefile

```diff
@@ -0,0 +1,51 @@
+find-dir-with = $(shell /usr/bin/perl -e 'chomp($$_ = `pwd`); while ($$_ ne "" && ! -e "$$_/$(1)") { m:(.*)/[^/]+/??:; $$_ = $$1; } print;')
```

Comment:
`BSG_MANYCORE_DIR := $(shell git rev-parse --show-toplevel)` might be simpler?

Reply:
Yup. Change is reflected in the recent commit.